src/resources/lang/de/crud.php

```diff
@@ -105,7 +105,7 @@
     'update_success' => 'Der Eintrag wurde erfolgreich geändert.',
 
     // CRUD reorder view
-    'reorder'                      => 'Reihenfolge',
+    'reorder'                      => 'Sortiere',
```

Comment:
Translation was not correct in the context it's used in.

Reply:
From the demo: 
![image](https://user-images.githubusercontent.com/1922624/51354362-adde3580-1ab3-11e9-8e07-35af66918f2c.png)